python/vowpalwabbit/pyvw.py

```diff
@@ -1847,18 +1851,121 @@ def iter_features(self):
                 v = self.feature_weight(ns, i)
                 yield f, v
 
-    def get_label(self, label_class=SimpleLabel):
-        """Given a known label class (default is Simplelabel), get
-        the corresponding label structure for this example.
+    def get_label(
+        self, label_class: Optional[Union[int, LabelType, AbstractLabel]] = None
+    ) -> Union[
+        "SimpleLabel",
+        "MulticlassLabel",
+        "CostSensitiveLabel",
+        "CBLabel",
+        "CCBLabel",
+        "SlatesLabel",
+        "CBContinuousLabel",
+    ]:
```

Comment:
Also would be nice if `AbstracLabel` can be used here

Reply:
If an AbstractLabel is returned then the specific functionality of each type is not accessible